src/components/Content/Content.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Wrapper from '../Wrapper';
+import ErrorBoundary from '../ErrorBoundary';
+import Filter from '../MovieFilter';
+import './style.css';
+
+const SearchResults = React.lazy(() => import('../SearchResults/SearchResults'));
```

Comment:
with index files you don't need to write like '../SearchResults/SearchResults', just '../SearchResults'

Reply:
Good note. Thanks. I fixed it and it will in a new PR.